README.md

```diff
@@ -1,2 +1,75 @@
-# turtle
-The fastest agile project management app
+# Turtle
+
+Turtle is an issue tracker for software developers.
+
+- easy to use (tasks are controlled by dragging and dropping them around)
+- built for agile development methodology
+- self-sufficient (does not require third-party services to function)
+- free and open-source
+
+Check it out at [https://turtly.co](https://turtly.co)
+
+## Getting started
+
+1. Clone the project
+
+  ```bash
+  git clone https://github.com/the-ninja-turtles/turtle.git -b develop && cd turtle
+  ```
+
+1. Configure environment variables in .env file
+
+  Create a .env file in the project root folder with an AUTH0_SECRET,
+  AUTH0_AUDIENCE, POSTGRES_USER and POSTGRES_PASSWORD.
+
+  ```bash
+  export AUTH0_SECRET=<secret>
+  export AUTH0_AUDIENCE=<audience>
+  export POSTGRES_USER=<user>
+  export POSTGRES_PASSWORD=<password>
+  ```
+  ```bash
+  source .env
+  ```
+
+1. Prerequisites
+
+  - [iojs 2.3.1][avn]
```

Comment:
avn only supports version switching, right? Why link to it then, and not to n/nvm?